com.ibm.team.build.hjplugin/pom.xml

```diff
@@ -99,12 +99,28 @@
 		</repository>
 	</repositories>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>commons-beanutils</groupId>
+				<artifactId>commons-beanutils</artifactId>
+				<!-- 1.9.4 when this merged https://github.com/stapler/stapler/pull/211 -->
```

Comment:
Copy-paste comment? It is not that trivial for a plugin, a core version would need to be updated

Reply:
just enforce it's the good version for digester3 we have here.

Reply:
actually digester3 only need 1.8.3 but I try to have something higher to avoid security scan trigger